src/Microsoft.SqlTools.ServiceLayer/Extensibility/IComposableService.cs

```diff
@@ -0,0 +1,16 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+
+namespace Microsoft.SqlTools.ServiceLayer.Extensibility
+{
+    /// <summary>
+    /// A Service that expects to lookup other services. 
+    /// </summary>
+    interface IComposableService
```

Comment:
Really should be public - will fix

---
In reply to: [101187599](https://github.com/Microsoft/sqltoolsservice/pull/229#discussion_r101187599) [](ancestors = 101187599)